packages-common/pluggable-widgets-generator/generators/app/templates/packages/package.js.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "<%- packageName %>",
+  "widgetName": "<%- widgetName %>",
+  "version": "<%- version %>",
+  "description": "<%- description %>",
+  "copyright": "<%- copyright %>",
+  "author": "<%- author %>",
+  "config": {
+    "widgetPath": "<%- projectPath %>/widgets",
+    "projectPath": "<%- projectPath %>/",
+    "mendixHost": "http://localhost:8080",
+    "developmentPort": "3000"
+  },
+  "packagePath": "<%- packagePath %>",
+  "scripts": {
+    "start": "concurrently \"pluggable-widgets-tools start:server --open\" \"npm:dev\"",
```

Comment:
We discussed that we don't need this `concurrently`. Only the start:server part.

Reply:
Changed it.